egs/wsj/s5/steps/nnet3/chain/get_egs.sh

```diff
@@ -66,6 +66,7 @@ max_shuffle_jobs_run=50  # the shuffle jobs now include the nnet3-chain-normaliz
                          # without overloading the disks.
 srand=0     # rand seed for nnet3-chain-copy-egs and nnet3-chain-shuffle-egs
 online_ivector_dir=  # can be used if we are including speaker information as iVectors.
+pick_random_ivector=false
```

Comment:
I'd rather just make this behavior the default.  It seems extremely unlikely that the old behavior would ever be better.  I know it's a hassle because it makes results not 100% comparable to old results.  But it will avoid having an extra configuration to deal with in the future.